webapp/locales/pt.json

```diff
@@ -252,7 +252,8 @@
     "newPost": "Criar uma nova publicação",
     "success": "Salvo!",
     "teaserImage": {
-      "cropperConfirm": "Confirmar"
+      "cropperConfirm": "Confirmar",
+      "supportedFormats": "Insira uma imagem do formato de arquivo JPG, PNG ou GIF"
```

Comment:
```json
"supportedFormats": "Insira uma imagem do formato JPG, PNG ou GIF"
```

Reply:
Please, accept my suggestion (make more sense say as I reviewed in Portuguese)

Reply:
Please, accept my suggestion (make more sense say as I reviewed in Portuguese)

Reply:
hey @ftonato  sure , i think a person from Portugal know it well that google translate :wink:

Reply:
Yeah, maybe (sometimes) ahaha
I'm actually from Brazil, but I live in Portugal, so this is a combination conhecimento de Portuguese o/

Reply:
This seems not resolved?
Or is it for you localy?